bindings_test.go

```diff
@@ -179,11 +179,11 @@ func TestSectorBuilderLifecycle(t *testing.T) {
 	})
 
 	// generate a PoSt
-	proofs, err := sb.GeneratePoSt(ptr, sectorInfo, [32]byte{}, []uint64{})
+	proofs, err := sb.GeneratePoSt(ptr, sectorInfo, [32]byte{}, []sb.Candidate{})
```

Comment:
@dignifiedquire - We should exercise the `GenerateCandidates` call in this test. I can update this PR if desired.